src/Twilio/TaskRouter/WorkflowRule.php

```diff
@@ -13,7 +13,7 @@ class WorkflowRule implements \JsonSerializable {
     public $friendly_name;
     public $targets;
 
-    public function __construct(string $expression, array $targets, string $friendly_name = null) {
+    public function __construct(string $expression, array $targets, ?string $friendly_name = null) {
```

Comment:
does this need the be questioned, also?

Reply:
Any typed parameter that defaults to null must have a nullable type.  PHP 7.1 uses ? as the nullable type modifier. Later versions would accept string|null, but not 7.1, so this is correct for PHP 8.4 to avoid a depreciation message and to future proof the package.

Reply:
Oh, did not answer completely. No, $expression does not default to null, so it is required and must be a string. Adding ? would mean you could pass null to it, and that is probably wrong.